### DISCUSSION TITLE
Why is `BottomBarPanel.openTab` private and how to open custom tabs?

### DISCUSSION THREAD

**Discussion by jan-david-fischbach** [redhat-developer/vscode-extension-tester#1678] [Q&A] [answered]
We are developing an extension providing custom tabs in the bottom bar. What is the intended way of interfacing with such tabs?
Regards JD

**Comment by pospisilf** [comment, marked as answer]
Hello Jan, thank you for bringing up this discussion. Your request was solved by https://github.com/redhat-developer/vscode-extension-tester/issues/1696 so now you can use `BottomBarPanel.openTab()` to open custom tabs. If you have any follow up questions, feel free to ask.

**Reply by jan-david-fischbach** [reply]
Thanks a lot
